network/dhcpd.py

```diff
@@ -0,0 +1,181 @@
+#!/usr/bin/env python
```

Comment:
set shebang to `#!/usr/bin/python`, also see http://docs.ansible.com/faq.html#how-do-i-handle-python-pathing-not-having-a-python-2-x-in-usr-bin-python-on-a-remote-machine